lib/generateClass.js

```diff
@@ -1084,70 +1084,70 @@ function preprocessSlicing(fhirProfile) {
         // starting a new slice element, only possible if already within a slice group
         sliceStack[sliceIndex].currentSlice = sliceNm;
         sliceStack[sliceIndex].elements[sliceNm] = [];
-      }
 
-      for (const sliceGroup of sliceStack) { // multiple slices can slice on the same discriminator so this needs to loop
-        if (pathMatchesDiscriminator(element.path, sliceGroup.slicing.discriminator)) {
-          // element.path matches a discriminator; we found a value to match on
+        for (const sliceGroup of sliceStack) { // multiple slices can slice on the same discriminator so this needs to loop
```

Comment:
All the code below is just whitespace changes because we moved it to be inside of the `if` statement above.  Previously it had been outside that `if` block so it was indented one level less than it is now.